.gitignore

```diff
@@ -2,7 +2,6 @@
 *.o
 *.a
 *.so
-*.mender
```

Comment:
Unrelated change

Reply:
without it I will not be able to add the artifacts.

Reply:
Yes, but it deserves a separate commit or a mention in the commit description.

Reply:
separate commit added